.github/workflows/tests.yml

```diff
@@ -26,15 +26,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.9, '3.10', 3.11]
+        python-version: [3.11]
         database: [sqlite, postgres, mysql]
         exclude:
           - database: mysql
-            python-version: 3.9
+            python-version: 3.11
           - database: mysql
             python-version: 3.11
           - database: sqlite
-            python-version: 3.9
+            python-version: 3.11
```

Comment:
Why do you exclude the version 3.11 / sqlite tests? 
The orginal coede had tests matrix 3.9 / 3.10 / 3..11  and only 3.9/sqlite was excluded.

I think all the exclude steps should be removed!!!

Reply:
fixed the changes